ringcentral/http/client.py

```diff
@@ -13,11 +13,14 @@ class Client:
     def __init__(self):
         pass
 
-    def send(self, request):
+    def send(self, request, multipart_mixed=False):
```

Comment:
This could be done in Request itself, no need to do this here. Method should not know what it sends. Moreover, you introduce this flag twice (here and in Platform) which is not good.